src/website/src/app/documentation/demos/header/header-links.demo.html

```diff
@@ -13,7 +13,7 @@
             </div>
             <div class="header-actions">
                 <a href="javascript://" class="nav-link nav-icon">
-                    <clr-icon shape="cog"></clr-icon>
+                    <clr-icon shape="cog" aria-label="settings"></clr-icon>
```

Comment:
Aria labels should ideally be on the parent interactive item `<a>` and `<button>`
https://clarity.design/icons/accessibility